Makefile

```diff
@@ -1,4 +1,4 @@
-DOCKER_RUN = DOCKER_BUILDKIT=1 docker-compose run php_client
+DOCKER_RUN ?= DOCKER_BUILDKIT=1 docker-compose run php_client
```

Comment:
This allows to override locally the environment variable with an export. In my case, I don't use `docker-compose` but `docker compose`.

Reply:
@samokiss your solution to be able to use Compose V2 :slightly_smiling_face:

Reply:
Another solution would be to replicate what JM did on the PIM (I think the PR is not merged yet) and what we did for now quite some time on Shared Catalogs: abandon Compose v1. Compose v2 is out for more than 2 years now, and Circle CI latests "machine" images work like a charm with it.

Reply:
Thank you, we have this problem on our project! And...--> https://docs.docker.com/compose/release-notes/#:~:text=From%20the%20end%20of%20June,from%20all%20Docker%20Desktop%20versions.